macros/get_flow_columns.sql

```diff
@@ -12,6 +12,21 @@
     {"name": "customer_filter", "datatype": dbt_utils.type_string()}
 ] %}
 
+/*******************
+  Did we previously settle on the reserved words to be handled within the macro so the staging
+  model doesn't have additional complexity? I checked our HubSpot `get_email_event_sent_columns` and 
+  noticed we handle this reserved words using the below in the macro. Then we are able to set "flow_trigger"
+  within the staging model without needing a conditional in the staging model. 
+
+  Both will work, but forgot what we decided was the approach to use?
+*******************
+{% if target.type == 'snowflake' %}
+ {{ columns.append({"name": "TRIGGER", "datatype": dbt_utils.type_string(), "quote": True, "alias": "flow_trigger"}) }}
+{% else %}
+ {{ columns.append({"name": "trigger", "datatype": dbt_utils.type_string(), "quote": True, "alias": "flow_trigger"}) }}
+{% endif %}
+*******************/
```

Comment:
i _think_ we settled on aliasing reserved words in the staging model (so that all the aliasing happens in one place), but i could  be wrong 🤔 @fivetran-joemarkiewicz

Reply:
Yeah that makes sense and I recently found a note from myself saying we will alias in the staging model so we will continue to use this method in our packages. Thanks!